### ISSUE TITLE
Renaming of a project doesn't work

### ISSUE THREAD

**Issue by JaroslavTulach** [enso-org/enso#7917]
I've just created new project locally. E.g. running:
```
 ./run gui watch --skip-wasm-opt
```
and
```
sbt runProjectManagerDistribution
```
and connecting to `localhost` `8080` via Chrome browser.

The newly created project got `New_Project_22` name. As soon as it opened, I returned back to dashboard and stopped the project. Then I renamed it to `SDG` and opened it again. Everything is broken, nothing works:

![Everything is broken](https://github.com/enso-org/enso/assets/26887752/4c882927-ef0f-475e-b233-571d9ec5fdb5)

Even the name of the project is still `New_Project_22`! The project is still in `New_Project_22` directory on disk, its `package.yaml` contains new name:
```
name: SDG
namespace: local
version: 0.0.1
license: ''
authors: []
maintainers: []
edition: 0.0.0-dev
prefer-local-libraries: true
```

After closing my browser, the project folder on disk renamed to `SDG`. Connecting again to localhost `8080` and opening the project after writing this bug report seems to get things into normal mode of operations.

**Comment by 4e6** [comment]
Renaming fails because the dashboard expects the engine version field in the response. Then the IDE is started with an incorrect (old) project name.

**Comment by somebody1234** [comment]
@JaroslavTulach i'm having issues running the old gui for some reason, so i'm not 100% sure it solves the issue, but a PR with a fix has been opened:
- #7975

**Comment by 4e6** [comment]
PR fixes the issue for me

**Comment by 4e6** [comment]
Fixed by #7975
